DomainContent/Trivia/FourSquareTrivia/triviaColorCheckBlue.js

```diff
@@ -0,0 +1,116 @@
+// triviaColorCheck.js
```

Comment:
I would probably combine all of these files and just check which color zone it is for if possible. Definitely not required and maybe not even an option though.

Reply:
not really an option.